### PR TITLE
add user story and incomplete test

### PR DESCRIPTION
**Pull request by odeac**
## Overview
add user story and incomplete test for wallet creation

### JIRA ticket:
RHOL-1102

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

File: docs/features.md [hunk 1]
```diff
@@ -163,3 +163,12 @@
...
 
 * TBD
+
+## Contracts
+### As a user, I want to be able to create a wallet so that I can store REV in it
+#### Create wallet
+##### test: test/test_wallet.py::test_create_wallet
+##### steps: 
+* send a message  to the process registered as 'rho:mint' with the structure **("create", pk, returnChannel)** where pk is the public key of the wallet
+* expect a response on the return channel with the unforgeable name of the wallet
```
Comment (KentShikama) [inline docs/features.md]: Is this `rho:id:<whatever>`?

File: docs/features.md [hunk 2]
```diff
@@ -163,3 +163,12 @@
 ##### steps:
 
 * TBD
+
+## Contracts
+### As a user, I want to be able to create a wallet so that I can store REV in it
+#### Create wallet
+##### test: test/test_wallet.py::test_create_wallet
+##### steps: 
+* send a message  to the process registered as 'rho:mint' with the structure **("create", pk, returnChannel)** where pk is the public key of the wallet
```
Comment (rabbitonweb) [inline docs/features.md]: please add empty line after steps, so that `"source of truth"` can parse it correctly

File: docs/features.md [hunk 3]
```diff
@@ -163,3 +163,12 @@
...
+* expect a response on the return channel with the unforgeable name of the wallet
+
```
Comment (rabbitonweb) [inline docs/features.md]: are there any other assertions we can do to check that the wallet exists?

File: docs/features.md (hunk 2 above)
Comment (dckc) [inline docs/features.md]: The mint can't go in the registry. If it did, then anyone in the world could mint REV.

What's registered is a contract for making empty rev purses.

In the current implementation, there's an indirection via SystemInstancesRegistry, which is a little tedious. but it works.

https://github.com/rchain/rchain/blob/a62dd9b443f00104277841ae34c7684c862fe496/casper/src/main/rholang/SystemInstancesRegistry.rho

See for example:
https://github.com/dckc/RChain-API/blob/cli/examples/payment/aliceBobWallets.rho d1bad4e

**Pull request closed without merging**